packages/roosterjs-editor-core/lib/coreApi/transformColor.ts

```diff
@@ -58,7 +58,7 @@ export const transformColor: TransformColor = (
     if (direction == ColorTransformDirection.DarkToLight) {
         transformToLightMode(elements);
     } else if (core.lifecycle.onExternalContentTransform) {
-        elements.forEach(element => core.lifecycle.onExternalContentTransform(element));
+        elements.forEach(element => core.lifecycle.onExternalContentTransform?.(element));
```

Comment:
We already checked null for core.lifecycle.onExternalContentTransform on line 60, so we can add "!" here if compiler still complains